timescaledb/getting-started/migrate-data.md

```diff
@@ -1,6 +1,6 @@
-# Migrating Data
+# Migrating data
```

Comment:
BTW, I realized that this content is copied lower in the tree. This (and a few other files) should be removed once this PR is merged as it's not actually linked from anywhere. (oh restructure... 😄 )

Reply:
Yeah, I think there are more than a few duplicates, actually. More for the to do list ...